btb/tuning/hyperparams/boolean.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+"""Package where the boolean hyperparameters are defined."""
+
+import numpy as np
+
+from btb.tuning.hyperparams.base import BaseHyperParam
+
+
+class BooleanHyperParam(BaseHyperParam):
+    """Boolean Hyperparameter Class.
+
+    The boolean hyperparameter class it is responsible for the transformation of boolean values in
+    to normalized search space of [0, 1]^K and also providing samples of those or the inverse
+    transformation from search space to hyperparameter space.
+
+    Hyperparameter space:
+        ``{True, False}``
+
+    Search space:
+        ``{0, 1}^K``
+
+    Attributes:
+        K (int):
+            Number of dimensions that this hyperparameter uses to be represented in the search
+            space.
+    """
+
+    K = 1
+
+    def _within_hyperparam_space(self, values):
+        if values.dtype is not np.dtype('bool'):
+            raise ValueError('Values: {} not within hyperparameter space.'.format(values))
+
+    def _inverse_transform(self, values):
+        """Invert one or more values from search space [0, 1]^K.
```

Comment:
Update the docstring to mention the actual search space `{0, 1}` instead of `[0, 1]^K`